<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormStatusExitProgram</class>
 <widget class="QWidget" name="FormStatusExitProgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сохранение данных..</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelMainMsg">
       <property name="text">
        <string>Field1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSecondMsg">
       <property name="text">
        <string>Field 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
